optuna/study.py

```diff
@@ -45,6 +45,7 @@
     from typing import Union  # NOQA
 
     from optuna.distributions import BaseDistribution  # NOQA
+    from optuna.samplers import BaseSampler  # NOQA
```

Comment:
Is there any new usage of `BaseSampler` in this file?

Reply:
No, it isn't. I will remove it.